src/handlers/updateMatch.ts

```diff
@@ -0,0 +1,62 @@
+import Joi from '@hapi/joi'
+import * as express from 'express-serve-static-core'
+
+import * as db from '../gateways/storage'
+import { ValidatedRequest } from '../middlewares/validator'
+
+export const schema = {
+  body: Joi.object<{
+    id: number
+    winnerId: number
+    victoryCondition: number
+    firstPlayerId?: number
+    deckARoleId?: number
+    deckBRoleId?: number
+    deckASplashId?: number
+    deckBSplashId?: number
+  }>({
+    id: Joi.number().required(),
+    winnerId: Joi.number().required(),
+    victoryCondition: Joi.number()
+      .min(1)
+      .required(),
+    firstPlayerId: Joi.number().optional(),
+    deckARoleId: Joi.number()
+      .min(1)
+      .optional(),
+    deckBRoleId: Joi.number()
+      .min(1)
+      .optional(),
+    deckASplashId: Joi.number()
+      .min(1)
+      .optional(),
+    deckBSplashId: Joi.number()
+      .min(1)
+      .optional(),
```

Comment:
We can enforce the numbers being integers like this:
```ts
Joi.number().integer().min(1).optional();
```

[Joi](Joi.number().integer().min(1).optional();) is very powerful. :)